balancer_pool_states.py

```diff
@@ -221,27 +241,26 @@ def compute_balancer_pool_states():
             balancer_pool_swaps[i]["block"]
             < balancer_pool_liquidity_changes[j]["block"]
         ):
-            current_WETH += balancer_pool_swaps[i]["WETH"]
-            current_COW += balancer_pool_swaps[i]["COW"]
-            balancer_pool_states.append(
-                {
+            CURRENT_TOKEN1 += balancer_pool_swaps[i][TOKEN1]
+            CURRENT_TOKEN2 += balancer_pool_swaps[i][TOKEN2]
+            balancer_pool_states.append({
                     "block": balancer_pool_swaps[i]["block"],
-                    "WETH": current_WETH,
-                    "COW": current_COW,
+                    TOKEN1: CURRENT_TOKEN1,
+                    TOKEN2: CURRENT_TOKEN2,
                     "time": balancer_pool_swaps[i]["time"],
-                }
-            )
-            i = i + 1
+                })
+            i = i + 1            
         else:
-            current_WETH += balancer_pool_liquidity_changes[j]["WETH"]
-            current_COW += balancer_pool_liquidity_changes[j]["COW"]
+            CURRENT_TOKEN1 += balancer_pool_liquidity_changes[j][TOKEN1]
+            CURRENT_TOKEN2 += balancer_pool_liquidity_changes[j][TOKEN2]
             balancer_pool_states.append(
                 {
                     "block": balancer_pool_liquidity_changes[j]["block"],
-                    "WETH": current_WETH,
-                    "COW": current_COW,
-                    "time": balancer_pool_swaps[i]["time"],
+                    TOKEN1: CURRENT_TOKEN1,
+                    TOKEN2: CURRENT_TOKEN2,
+                    "time": balancer_pool_liquidity_changes[j]["time"],
                 }
             )
-            j = j + 1
+            j = j + 1       
```

Comment:
Run black once to get rid of trailing white spaces.
Then run pylint for more stylistic suggestions.